### ISSUE TITLE
Exit status 1 error

### ISSUE THREAD

**Issue by nate-double-u** [mikerhodes/github-to-omnifocus#14]
I'm sometimes running across this error, but I'm having some trouble tracking down why. Is there a debug mode I can turn on to help see what's happening here?

```
2023/06/08 11:30:52 [main] Starting github2omnifocus; version: v2.4.
2023/06/08 11:30:52 Config loaded from /Users/USERNAME/.config/github2omnifocus/config.json:
2023/06/08 11:30:52   GitHub API server: https://api.github.com
2023/06/08 11:30:52   GitHub token: *****
2023/06/08 11:30:52   Omnifocus tag: github
2023/06/08 11:30:52   Omnifocus assigned issue project: GitHub Assigned
2023/06/08 11:30:52   Omnifocus PR to review project: GitHub Reviews
2023/06/08 11:30:52   Omnifocus notifications project: GitHub Notifications
2023/06/08 11:31:22 exit status 1
```

**Comment by nate-double-u** [comment]
I checked https://www.githubstatus.com/ to see if there may be an issue on GitHub's side, but they appear to have a green board.

<img width="1392" alt="Screenshot 2023-06-08 at 12 00 23 PM" src="https://github.com/mikerhodes/github-to-omnifocus/assets/4453979/2f0c123e-2cab-4b82-a9e8-a78ae445578a">

**Comment by mikerhodes** [comment]
@nate-double-u Ouch, I didn't see this issue. I'm going to guess it either fixed itself or you moved on to something that had a more active maintainer, but let me know if this is still something that is of interest?

**Comment by mikerhodes** [comment]
Closing, but leaving the investigation here if needed in the future.

I had a quick read of the code to see if I could figure where `2023/06/08 11:31:22 exit status 1` came from. It's not obvious. As this is a log line rather than a panic, I'm guessing that it comes from the code we use to interact with Omnifocus, where we run a JXA script there's some error case I don't account for in the JS code, and this bubbles up to hit us at:

https://github.com/mikerhodes/github-to-omnifocus/blob/35b6a27d5ff87cc4bb14d77c3c4d8d75c695fe6a/cmd/github2omnifocus/main.go#L67-L70

I'm guessing here because otherwise we'd have hit the log lines right below it that output the current and desired states.

This doesn't surprise me, because I found the JXA interface for Omnifocus pretty hard to work with, and hard to debug what was going on. This afflicted file is [this one](https://github.com/mikerhodes/github-to-omnifocus/blob/master/internal/omnifocus/jxa/oftasksforprojectwithtag.js) but I'm not really sure where any log lines or crash output it might generate would ever go.

So overall, pleased it started working 😬
